launcher/tests/test_notify.py

```diff
@@ -0,0 +1,247 @@
+import datetime
+import os
+import pytest
+import re
+import subprocess
+
+from unittest import mock
+from importlib.machinery import SourceFileLoader
+from multiprocessing import Pool
+from tempfile import TemporaryDirectory
+
+relpath_notify = "../sdw_notify/Notify.py"
+path_to_notify = os.path.join(os.path.dirname(os.path.abspath(__file__)), relpath_notify)
+notify = SourceFileLoader("Notify", path_to_notify).load_module()
```

Comment:
This is the pattern for relative imports that was already used in `sdw-launcher`, updated here to avoid the deprecation warning caused by the use of `imp`. However, I'm sure a cleaner relative import is possible and would be happy to refactor both files - pointers appreciated!

Reply:
The path insertions aren't ideal. Since we have an RPM package now, we can install the Python modules at the system level and import them directly. Let's not do that now, especially with #450 still pending review. Not critical pre-pilot, but would make the updater/notifier code much more maintainable.

Reply:
Yeah I agree. Note that this is just copying the approach that was already in place (slightly revised to avoid a deprecation warning for `imp`), but we should definitely clean the launcher organization up a bit.